src/dx/pubnub_client.rs

```diff
@@ -348,6 +349,7 @@ impl<T> PubNubClientConfigBuilder<T> {
             .map_err(|err| PubNubError::ClientInitialization(err.to_string()))
             .and_then(|pre_build| {
                 let token = Arc::new(spin::RwLock::new(String::new()));
+                debug!("Client Configuration: \n publish_key: {:?}\n subscribe_key: {}\n user_id: {}\n instance_id: {:?}", pre_build.config.publish_key, pre_build.config.subscribe_key, pre_build.config.user_id, pre_build.instance_id);
```

Comment:
Why not just debug print the pre_build.config? ;o

Reply:
or the whole client?

Reply:
it may include `secret_key` or `auth_key` if user provides it. we should not reveal it.

Reply:
and so I thought of being explicit about fields

Reply:
we never ask user for those values AFAIK

Reply:
ahh. Okay - I misunderstood that at DM.